src/sdk/src/fragment-components/a-slider.tsx

```diff
@@ -0,0 +1,240 @@
+		componentObj={componentObj}
+		select={select}
+		remove={remove}
+		selected={selected} />,
+	keywords: ['slider'],
+	name: 'Slider',
+	type: 'slider',
+	defaultComponentObj: {
+		type: 'slider',
+		labelText: 'Slider label',
+		min: 0,
+		max: 100
+	},
+	image,
+	codeExport: {
+		angular: {
+			inputs: ({ json }) => `@Input() ${nameStringToVariableString(json.codeContext?.name)}Label = "${json.labelText}";
+				@Input() ${nameStringToVariableString(json.codeContext?.name)}Disabled = ${json.disabled ? json.disabled : false};
+				@Input() ${nameStringToVariableString(json.codeContext?.name)}Value = ${json.value ? json.value : json.min};
+				@Input() ${nameStringToVariableString(json.codeContext?.name)}Min = ${json.min};
+				@Input() ${nameStringToVariableString(json.codeContext?.name)}Max = ${json.max};
+				@Input() ${nameStringToVariableString(json.codeContext?.name)}MinLabel = "${json.minLabel ? json.minLabel: ''}";
+				@Input() ${nameStringToVariableString(json.codeContext?.name)}MaxLabel = "${json.maxLabel ? json.maxLabel: ''}";
+				@Input() ${nameStringToVariableString(json.codeContext?.name)}Step = ${json.step ? json.step : 1};
+				@Input() ${nameStringToVariableString(json.codeContext?.name)}StepMultiplier = ${json.stepMultiplier ? json.stepMultiplier : 1};
+				@Input() ${nameStringToVariableString(json.codeContext?.name)}Theme = "${json.light ? 'light' : 'dark'}" `,
+			outputs: ({ json }) => `@Output() ${nameStringToVariableString(json.codeContext?.name)}ValueChange = new EventEmitter();`,
+			imports: ['SliderModule'],
+			otherImports: () => 'import { CommonModule } from "@angular/common"',
+			code: ({ json }) => {
+				return `<ibm-slider
+					[label]="${nameStringToVariableString(json.codeContext?.name)}Label"
+					[min]="${nameStringToVariableString(json.codeContext?.name)}Min"
+					[max]="${nameStringToVariableString(json.codeContext?.name)}Max"
+					[step]="${nameStringToVariableString(json.codeContext?.name)}Step"
+					[value]="${nameStringToVariableString(json.codeContext?.name)}Value"
+					[shiftMultiplier]="${nameStringToVariableString(json.codeContext?.name)}StepMultiplier"
+					[disabled]="${nameStringToVariableString(json.codeContext?.name)}Disabled"
+					(valueChange)="${nameStringToVariableString(json.codeContext?.name)}ValueChange.emit($event.value)">
+					<span minLabel>{{${nameStringToVariableString(json.codeContext?.name)}MinLabel}}</span>
+					<span maxLabel>{{${nameStringToVariableString(json.codeContext?.name)}MaxLabel}}</span>
+                    <input theme="${nameStringToVariableString(json.codeContext?.name)}Theme" 
+                    ${angularClassNamesFromComponentObj(json)} />
+				</ibm-slider>`;
+			}
+		},
+		react: {
+			imports: ['Slider'],
+			code: ({ json }) => {
+				return `<Slider
+					labelText='${json.labelText}'
+					min={${json.min}}
+					max={${json.max}}
+					value={${json.value}}
+					step={${json.step}}
+					stepMultiplier={${json.stepMultiplier}}
```

Comment:
```
					value={${json.value}}
					step={${json.step}}
					stepMultiplier={${json.stepMultiplier}}
```

It's undefined in exports when user doesn't specify the value. Let's reduce code here, if the user does not specify the value,step, or stepMultiplier, we should not include the respective prop.